indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -42,10 +42,15 @@ <h6>
     </p>
   {% endif %}
 
-  {% if work.commencement_date %}
-    <p class="mb-2">Commenced on {{ work.commencement_date|date:"Y-m-d" }}{% if work.commencing_work %} by
-      <a href="{% url 'work' frbr_uri=work.commencing_work.frbr_uri %}">
-        {{ work.commencing_work.title }}</a>{% endif %}.
+  {% if work.commenced %}
+    <p class="mb-2">
+      {% if work.commencements.count > 1 %}
+        This work has <a href="{% url 'work_commencements' frbr_uri=work.frbr_uri %}">multiple commencements</a>.
+      {% else %}
```

Comment:
If we have a main commencement date, we should always show it. We can then supplement that with the multiple commencements notice.